backend/src/test/java/codezap/template/service/facade/CategoryTemplateApplicationServiceTest.java

```diff
@@ -0,0 +1,131 @@
+package codezap.template.service.facade;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Collections;
+import java.util.List;
+
+import jakarta.transaction.Transactional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import codezap.category.domain.Category;
+import codezap.fixture.MemberFixture;
+import codezap.fixture.TemplateFixture;
+import codezap.global.ServiceTest;
+import codezap.global.exception.CodeZapException;
+import codezap.member.domain.Member;
+import codezap.template.domain.SourceCode;
+import codezap.template.domain.Template;
+import codezap.template.domain.Thumbnail;
+import codezap.template.dto.request.CreateSourceCodeRequest;
+import codezap.template.dto.request.CreateTemplateRequest;
+import codezap.template.dto.request.UpdateSourceCodeRequest;
+import codezap.template.dto.request.UpdateTemplateRequest;
+
+@Transactional
+class CategoryTemplateApplicationServiceTest extends ServiceTest {
+
+    @Autowired
+    private CategoryTemplateApplicationService categoryTemplateApplicationService;
+
+    @Nested
+    @DisplayName("템플릿 생성")
+    class CreateTemplate {
+
+        @Test
+        @DisplayName("성공: 작성자의 권한 확인 후 템플릿 생성")
+        void createTemplate_Success() {
+            // given
+            Member member = memberRepository.save(MemberFixture.getFirstMember());
+            Category membersCategory = categoryRepository.save(new Category("Members", member));
+            List<CreateSourceCodeRequest> sourceCodeRequests = List.of(
+                    new CreateSourceCodeRequest("filename", "content", 1));
+            CreateTemplateRequest request = new CreateTemplateRequest(
+                    "Template", "Description", sourceCodeRequests, 1, membersCategory.getId(), List.of("태그"));
+
+            // when
+            Long result = categoryTemplateApplicationService.createTemplate(member, request);
+
+            // then
+            assertThat(result).isEqualTo(1L);
```

Comment:
확인하고 싶은 것이 "저장된 템플릿의 아이디가 1L 이다" 보단, 원하는 대로 잘 저장되었음을 확인하고 싶은 것이니..
아래와 같이 바꾸어 보는 것은 어떨까요?
```suggestion
            assertThat(categoryRepository.existsById(result)).isTrue();
```